templates/app/cpp/tizen-manifest-service.xml.tmpl

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="{{androidIdentifier}}" version="1.0.0" api-version="4.0" xmlns="http://tizen.org/ns/packages">
+    <profile name="common"/>
+    <service-application appid="{{androidIdentifier}}" exec="runner" type="capp" multiple="false" nodisplay="true" taskmanage="false" auto-restart="false">
+        <label>{{projectName}}</label>
+        <icon>ic_launcher.png</icon>
+        <background-category value="media"/>
```

Comment:
According to [this](https://developer.tizen.org/ko/development/tizen-studio/web-tools/configuring-your-app/configuration-editor?langredirect=1) besides of adding background-category field there should be also a <tizen:setting background support="enable" /> to make the behaviour correct. But as I am not tizen-manifest expert, thus maybe this is not true for service applications.

Reply:
AFAIK native application does not have `background support` selector. And it looks like `background-category` is enough.